modules/express-engine/schematics/install/files/server-tools/add-ssl.ts

```diff
@@ -0,0 +1,71 @@
+import * as express from 'express';
+import { readFileSync } from 'fs';
+import * as https from 'https';
+import { ServerSettings } from './server-settings.interface';
+
+
+// Webpack will replace 'require' with '__webpack_require__'
+// '__non_webpack_require__' is a proxy to Node 'require'
+declare const __non_webpack_require__: NodeRequire;
+
+
+
+/**
+ * Add SSL support to an express server.
+ * It uses the nodes https.createServer to wrap the express server
+ * It uses the (lazy-loaded) selfcert package to generate
+ * a self-signed certificate on the fly if there isn't one provided.
+ * @param server the express app/server
+ */
+export function addSSL(server: express.Express,serverSettings:ServerSettings): express.Express {
+  const pems = {
+    private: '',
+    cert: '',
+  };
+  if (serverSettings.sslCert && serverSettings.sslKey) {
+    try {
+      pems.private = readFileSync(serverSettings.sslKey).toString();
+      pems.cert = readFileSync(serverSettings.sslCert).toString();
+    } catch (e) {
+      console.log(`
+        Please check the path for the certificate.
```

Comment:
This may need to be an error, since you are calling `process.exit` afterwards. Also, the error should mention the the `sslKey` and `sslCert` as well.